### PR TITLE
update schema-registry.md

### PR DESCRIPTION
**Pull request by HAHAZHUZHU**

### CODE REVIEW

File: en_US/rule/schema-registry.md [hunk 1]
```diff
@@ -1,89 +1,87 @@
...
 
 
-## 数据格式
+## Data Format
 
-下图展示了 Schema Registry 的一个应用案例。多个设备上报不同格式的数据，经过 Schema Registry 解码之后，变为统一的内部格式，然后转发给后台应用。
+The diagram below shows an example of a Schema Registry application. Multiple devices report data in different formats, which are decoded by Schema Registry into a uniform internal format and then forwarded to the backend application.
 
-![schema-registry](./assets/schema-registry.png)
+![schema-registry](/Users/liurz/工作/翻译文件/EMQ X 消息服务器简介/assets/schema-registry.png)
```
Comment (sigming) [inline en_US/rule/schema-registry.md]: Please place the image in `en_US/rule/assets/`, don't change the code

File: en_US/rule/schema-registry.md [hunk 2]
```diff
@@ -1,89 +1,87 @@
...
+## Architecture Design
 
-Schema Registry 为 Avro 和 Protobuf 等内置编码格式维护 Schema 文本，但对于自定义编解码 (3rd-party) 格式，如需要 Schema，Schema 文本需由编解码服务自己维护。Schema API 提供了通过 Schema Name 的添加、查询和删除操作。
+Schema Registry maintains Schema text for built-in encoding formats such as Avro and Protobuf, but for custom codec (3rd-party) formats, Schema text needs to be maintained by the codec service itself, if required. The Schema API provides for add, query, and delete operations via Schema Name.
 
-Schema Registry 既可以解码，也可以编码。编码和解码时需要指定 Schema Name。
+The Schema Registry can perform both decoding and encoding.  Schema Name needs to be specified when encoding and decoding.
 
-![architecture](./assets/arch.png)
+![architecture](/Users/liurz/工作/翻译文件/EMQ X 消息服务器简介/assets/arch.png)
```
Comment (sigming) [inline en_US/rule/schema-registry.md]: `./assets/arch.png`

File: en_US/rule/schema-registry.md [hunk 3]
```diff
@@ -1,89 +1,87 @@
...
+## Codec + Rules Engine
 
-EMQ X 的消息处理层面可分为消息路由(Messaging)、规则引擎(Rule Engine)、数据格式转换(Data Conversion) 三个部分。
+The message processing level of EMQ X can be divided into three parts: Messaging, Rule Engine, and Data Conversion.
 
-EMQ X 的 PUB/SUB 系统将消息路由到指定的主题。规则引擎可以灵活地配置数据的业务规则，按规则匹配消息，然后指定相应动作。数据格式转换发生在规则匹配的过程之前，先将数据转换为可参与规则匹配的 Map 格式，然后进行匹配。
+EMQ X's PUB/SUB system routes messages to specified topics. The rule engine has the flexibility to configure business rules for the data, match messages to the rules and then specify the corresponding action. Data format conversion occurs before the rule matching process, converting the data into a Map format that can participate in rule matching, and then matching it.
 
-![SchemaAndRuleEngine](./assets/SchemaAndRuleEngine.png)
+![SchemaAndRuleEngine](/Users/liurz/工作/翻译文件/EMQ X 消息服务器简介/assets/SchemaAndRuleEngine.png)
```
Comment (sigming) [inline en_US/rule/schema-registry.md]: `./assets/SchemaAndRuleEngine.png`

**Review by tigercl — APPROVED** [review]

**Review by sigming — APPROVED** [review]

File: en_US/rule/schema-registry.md [hunk 4]
```diff
@@ -1,89 +1,87 @@
-# 编解码（Schema Registry）介绍
+# Introduction to Codec (Schema Registry)
```
Comment (terry-xiaoyu) [inline en_US/rule/schema-registry.md]: Introduction to Schema Registry

Comment (HAHAZHUZHU) [reply en_US/rule/schema-registry.md]: Already fixed @terry-xiaoyu

**Pull request merged**